proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java

```diff
@@ -58,28 +59,34 @@
 import io.getlime.security.powerauth.keychain.fingerprint.IFingerprintActionHandler;
 import io.getlime.security.powerauth.networking.client.HttpClient;
 import io.getlime.security.powerauth.networking.client.JsonSerialization;
+import io.getlime.security.powerauth.networking.endpoints.ConfirmRecoveryCodeEndpoint;
 import io.getlime.security.powerauth.networking.endpoints.CreateActivationEndpoint;
 import io.getlime.security.powerauth.networking.endpoints.RemoveActivationEndpoint;
 import io.getlime.security.powerauth.networking.endpoints.ValidateSignatureEndpoint;
 import io.getlime.security.powerauth.networking.endpoints.VaultUnlockEndpoint;
 import io.getlime.security.powerauth.networking.interfaces.ICancelable;
 import io.getlime.security.powerauth.networking.interfaces.INetworkResponseListener;
+import io.getlime.security.powerauth.networking.model.entity.ActivationRecovery;
+import io.getlime.security.powerauth.networking.model.request.ActivationLayer1Request;
+import io.getlime.security.powerauth.networking.model.request.ConfirmRecoveryRequestPayload;
+import io.getlime.security.powerauth.networking.model.response.ConfirmRecoveryResponsePayload;
 import io.getlime.security.powerauth.networking.response.CreateActivationResult;
 import io.getlime.security.powerauth.networking.response.IActivationRemoveListener;
 import io.getlime.security.powerauth.networking.response.IActivationStatusListener;
 import io.getlime.security.powerauth.networking.response.IAddBiometryFactorListener;
 import io.getlime.security.powerauth.networking.response.IChangePasswordListener;
+import io.getlime.security.powerauth.networking.response.IConfirmRecoveryCodeListener;
 import io.getlime.security.powerauth.networking.response.ICreateActivationListener;
 import io.getlime.security.powerauth.networking.response.IDataSignatureListener;
 import io.getlime.security.powerauth.networking.response.IFetchEncryptionKeyListener;
+import io.getlime.security.powerauth.networking.response.IGetRecoveryDataListener;
 import io.getlime.security.powerauth.networking.response.IValidatePasswordListener;
-import io.getlime.security.powerauth.rest.api.model.entity.ActivationType;
-import io.getlime.security.powerauth.rest.api.model.request.v3.ActivationLayer1Request;
-import io.getlime.security.powerauth.rest.api.model.request.v3.ActivationLayer2Request;
-import io.getlime.security.powerauth.rest.api.model.request.v3.VaultUnlockRequestPayload;
-import io.getlime.security.powerauth.rest.api.model.response.v3.ActivationLayer1Response;
-import io.getlime.security.powerauth.rest.api.model.response.v3.ActivationLayer2Response;
-import io.getlime.security.powerauth.rest.api.model.response.v3.VaultUnlockResponsePayload;
+import io.getlime.security.powerauth.networking.model.entity.ActivationType;
```

Comment:
It's good to "Code -> Optimize Imports" in Android Studio. It'll sort import lexicographically.
The same in other files where the package name has been changed.

Reply:
Fixed